ZConfig/_schema_utils.py

```diff
@@ -308,15 +308,32 @@ def _visit_default(self, name, info):
     del TypeVisitor
 
 
-def load_schema(schema, package, package_file):
+def load_schema(schema,
+                package=None):
+    """
+    Load the *schema* and return the schema object.
+
+    By default, *schema* is interpreted as a path on disk to a schema
+    file.
+
+    If *package* is set to a true string value, then *package* must
```

Comment:
Maybe "non-empty string value" instead of a "true string value"?

(When I see "true string value" I don't know if the software expects `"true"`/`"false"` or nonblank/blank.)